doc/zh-cn/plugins/limit-count.md

```diff
@@ -44,6 +44,7 @@
 | redis_host      | string   | `redis` 必须 |         |                                                              | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的地址。   |
 | redis_port      | integer  | 可选         | 6379    | [1,...]                                                      | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的端口     |
 | redis_password  | string   | 可选         |         |                                                              | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点的密码。   |
+| redis_database  | integer  | 可选         | 0       | redis_database >= 0                                          | 当使用 `redis` 限速策略时，该属性是 Redis 服务节点中使用的 database，并且只针对 非Redis集群模式（单实例模式 或者 提供单入口的Redis公有云服务）生效。   |
```

Comment:
Some minor style problems: Keep empty space on both sides of English words. Remove redundant empty spaces between Chinese words.

Reply:
Uh, yes, I modified the related grammar, thank you very much for reminding